### PR TITLE
Setup basic application server

### PR DESCRIPTION
**Pull request by ndt93**
Set up basic Hapi server/routes, UserController, testing, monitoring, logging and documentation framework.

### CODE REVIEW

File: app_server/app/Router.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+var logger = Utility.createLogger(__filename);
+
+/* Configure Hapi server connection */
+var server = new Hapi.Server();
+server.connection({ port: 3000 });
+server.route({
+  method: 'GET',
+  path: '/',
+  handler: function (request, reply) {
+    reply("Welcome to WorldScope");
```
Comment (yj-soh) [inline app_server/app/Router.js]: `'` instead of `"`?
Likewise for the rest of this file and `test/local/Utility.js`.

**Comment by yj-soh** [comment]
Seems fine aside from that.

Is that comment format in `Utility.js` the convention / common?
This thing:

```
var abc =
/**
 * Comment line 1 here
 * Comment line 2 here
 */
exports.abc = function(def) {
  ...
};
```

It feels odd inserting a multiline comment block in the middle like that.

**Comment by ndt93** [comment]
Since I'm not implementing `Utility` as a "class", It's one of the trick in `jsdoc` to make it documents the function as a exported static function while making the function available inside the module without documenting it twice. http://usejsdoc.org/howto-commonjs-modules.html

It doesn't decrease readability in my opinion, but if you think there is a better convention then we can change that. Also most other modules beside `Utility` should be implemented as one `constructor` a.k.a "class".

File: app_server/app/Router.js (hunk 1 above)
Comment (ndt93) [inline app_server/app/Router.js]: Ok, sure. I will change this to be more consistent.

**Comment by ndt93** [comment]
@yj-soh I fixed the quotes thing, can merge?

**Comment by ndt93** [comment]
I've also changed to using `rfr` module instead of using relative paths for requiring modules.

**Comment by yj-soh** [comment]
Looks alright.
If that comment format is suggested by the library, go with it.
I'm just concerned because I hadn't seen it before.

**Pull request merged**